java/org/apache/catalina/connector/ResponseFacade.java

```diff
@@ -111,23 +111,19 @@ public String getCharacterEncoding() {
 
     @Override
     public ServletOutputStream getOutputStream() throws IOException {
-        checkFacade();
-        ServletOutputStream sos = response.getOutputStream();
         if (isFinished()) {
             response.setSuspended(true);
         }
-        return sos;
+        return response.getOutputStream();
     }
 
 
     @Override
     public PrintWriter getWriter() throws IOException {
-        checkFacade();
-        PrintWriter writer = response.getWriter();
         if (isFinished()) {
             response.setSuspended(true);
         }
-        return writer;
+        return  response.getWriter();
```

Comment:
There is an extra space here that needs to be removed.

Reply:
@markt-asf I removed that extra space! Thank you